sql/appointment_opd_speciality.sql

```diff
@@ -0,0 +1 @@
+INSERT INTO appointment_speciality (name, date_created, creator, uuid) VALUES ('APPOINTMENT_OPD_SPECIALITY_KEY', NOW(), 1, UUID());
```

Comment:
@rubailly Could you please hard code the uuid here? The reason for this is to have the same uuid across all our environments; this makes it easier to reference the record in the future if we need to modify it. You can generate a uuid from: https://www.uuidgenerator.net/

Reply:
@tmvumbi2 -Good point. I'll hardcode the uuid and get back to you.

Reply:
@tmvumbi2 -Good point. I'll hardcode the uuid and get back to you.